packages/pwa/app/utils/site-utils.js

```diff
@@ -58,7 +58,7 @@ export const getSiteByHostname = (hostname) => {
         throw new Error('No site config found. Please check you configuration')
     if (!hostname) return
 
-    const site = sites.filter((site) => site.hostnames.includes(hostname))
+    const site = sites.filter((site) => site?.hostnames?.includes(hostname))
```

Comment:
The logic that you want to have here should probably be something like this

```
const site = sites.filter((site) => site.hostnames && site.hostnames.includes(hostname))
```

The reason why it should be like this is because using the optional syntax at the point of calling `includes` coincidentally give you the desired outcome. Imaging you were filtering the inverse (getting a list of all the sites that doesn't include a given hostname) in this if `hostnames` was undefined it would give a false positive.